package.json

```diff
@@ -41,6 +41,25 @@
             "title": "Prometheus URL",
             "description": "Enter the base URL for your prometheus instance, this URL is used to generate links for queries",
             "default": "http://localhost:9090"
+          },
+          "autometrics.experimentalRustSupport": {
+            "type": "boolean",
+            "title": "Enable experimental Rust support",
```

Comment:
do the title props do anything here? AFAICT the title for properties comes from its key, and i don't see titles mentioned [here](https://code.visualstudio.com/api/references/contribution-points#contributes.configuration)

Reply:
You're right these titles were suggested by my editor. But the values being displayed in the UI are based on the name of the value.